code/src/components/css/moviebio.css

```diff
@@ -0,0 +1,136 @@
+.bio-section {
+  display: flex;
+  color: white;
+  align-items: center;
+  position: relative;
+  z-index: 0;
+}
+
+.background-image {
+  position: relative;
```

Comment:
This property doesn't really do anything. If we use `position: relative`, we usually want to set it children as `position: absolute` or change top/right/bottom/left properties. It's none of your cases, so we can delete it :)